ports/qemu-arm/README.md

```diff
@@ -15,7 +15,7 @@ The purposes of this port are to enable:
     - no need to use OpenOCD or anything else that might slow down the
       process in terms of plugging things together, pressing buttons, etc.
 
-This port will only work with with [GCC ARM Embedded](launchpad.net/gcc-arm-embedded)
+This port will only work with with [GCC ARM Embedded](https://launchpad.net/gcc-arm-embedded)
```

Comment:
According to this link:

> As previously announced all new binary and source packages will not be released on Launchpad henceforth, they can be found on:
https://developer.arm.com/open-source/gnu-toolchain/gnu-rm.

So it is probably better to just replace this with the new link.

Reply:
Done